autoreject/tests/test_autoreject.py

```diff
@@ -88,9 +88,15 @@ def test_autoreject():
         mne.pick_types(epochs.info, meg=False, eeg=True)[::16],
         mne.pick_types(epochs.info, meg=False, eeg=False, eog=True)]
     pick_ch_names = [epochs.ch_names[pp] for pp in pre_picks]
+    bad_ch_names = [epochs.ch_names[ix] for ix in range(len(epochs.ch_names))
+                    if ix not in pre_picks]
```

Comment:
This looks weird Shouldn't the bad channels be a subset of the channels in the epochs? Right now, `bad_ch_names` is not a subset of `epochs.info['ch_names']`

Reply:
bad_ch_names is a subset of epochs_with_bads.info['ch_names']. epochs_with_bads and epochs have the same good channels, but I mark the channels that are dropped in epochs as bad in epochs_with_bads rather than drop them. This way I can compare autoreject on the same set of good channels with and without other bad channels in the epochs to make sure the bad channels do not affect the result.

Reply:
yes, understood. I think I was just surprised by 360 bad channels :-)